src/model/operation/transform.js

```diff
@@ -71,47 +112,142 @@ export function transform( a, b, context = {} ) {
 	}
 }
 
+/**
+ * Performs a transformation of two sets of operations - `operationsA` and `operationsB`. The transformation is two-way -
+ * both transformed `operationsA` and transformed `operationsB` are returned.
+ *
+ * Note, that the first operation in each set should base on the same context. A simplified version of context is
+ * {@link module:engine/model/document~Document#version document version}, so the
```

Comment:
> A simplified version of context is document version.

It might be too abstract.

Can you just say:
`Note, that the first operation in each set should base on the same document state ({@link module:engine/model/document~Document#version document version}).`
?